includes/omise-wc-setting.php

```diff
@@ -0,0 +1,100 @@
+<?php
+defined( 'ABSPATH' ) or die( "No direct script access allowed." );
+
+return array(
```

Comment:
@nimid I'm still thinking that what's gonna happen if we remove `Omise` text-domain and someone already downloaded our plugin and did implement I18N feature by themselves,

their implementation'd break because their translation can't refer to our previous text-domain?